lib/src/browser/disposable_browser.dart

```diff
@@ -198,14 +199,20 @@ class Disposable implements disposable_common.Disposable {
   void manageDisposable(disposable_common.Disposable disposable) =>
       _disposable.manageDisposable(disposable);
 
+  @deprecated
   @override
   void manageDisposer(disposable_common.Disposer disposer) =>
       _disposable.manageDisposer(disposer);
 
   @override
+  SimpleDisposable getManagedDisposer(disposable_common.Disposer disposer) =>
+      _disposable.getManagedDisposer(disposer);
+
+  @override
   void manageStreamController(StreamController controller) =>
       _disposable.manageStreamController(controller);
 
+  @deprecated
   @override
   void manageStreamSubscription(StreamSubscription subscription) =>
```

Comment:
The doc comment with migration instructions should be inherited from [here](https://github.com/Workiva/w_common/blob/master/lib/src/common/disposable_manager.dart#L57)